libclamav_rust/logging.rs

```diff
@@ -0,0 +1,58 @@
+/* rust logging module */
+
+use std::ffi::c_void;
+use std::ffi::CString;
+use std::os::raw::c_char;
+
+extern crate log;
+//use log::debug;
+
+use self::log::{Level, Metadata, Record};
+use self::log::{LevelFilter};
```

Comment:
I was playing around with this a little bit and noticed that when I saved the file, my vscode format-on-save option made a few minor changes to this file.

I would like to add a github action in the future to check that `rustfmt` doesn't find any issues.  To start things off clean, can I trouble you to have vscode/rustfmt format this file? It would save time in the future too if you enable format-on-save so it doesn't have to be a conscious decision.